### PR TITLE
fixed location of i button and languages

### PR DESCRIPTION
**Pull request by Youngjun333**

### CODE REVIEW

File: source/scripts/accessibility.js [hunk 1]
```diff
@@ -105,6 +105,9 @@ function keyControls (e) {
     case keys.N:
       document.getElementById(RESET_NO_ID).click();
       break;
+    case keys.Q:
+      document.getElementById('info-open-button').click();
+      break;
```
Comment (jinbest17) [inline source/scripts/accessibility.js:110]: great work adding keyboard shortcut for tutorial page, we need to reflect this in the readme

File: source/scripts/util/language.js [hunk 2]
```diff
@@ -40,8 +40,8 @@ const enUS = {
   background2: 'Desert',
   background3: 'Lake',
   dropdownEn: 'English',
-  dropdownKo: 'Korean',
-  dropdownEs: 'Spanish',
+  dropdownKo: '한국어',
+  dropdownEs: 'Español',
```
Comment (jinbest17) [inline source/scripts/util/language.js:44]: great job fixing language dropdown

**Review by jinbest17 — APPROVED** [review]
Great job!

File: build/tutorial_page.html [hunk 3]
```diff
@@ -96,7 +96,7 @@
                     <!-- Staggered Description Elements -->
```
Comment (friidryce) [inline build/tutorial_page.html:96]: Good catch of updating the descriptions

**Review by friidryce — APPROVED** [review]
Good work on noticing the details!

**Pull request merged**